embulk-core/src/main/java/org/embulk/exec/PreviewExecutor.java

```diff
@@ -36,6 +36,10 @@
     public interface PreviewTask
             extends Task
     {
+        @Config("exec")
+        @ConfigDefault("{}")
+        public ConfigSource getExecConfig();
```

Comment:
Any reason why we use `exec` for this while `preview_sample_rows` is out of `exec`?

Sorry, I was not very sure how `preview_sample_rows` is configured. Is it out of `exec`? In other words, the configurations is like following?

```
preview_sample_rows: 50
exec:
  preview_sample_buffer_bytes: 10000
in:
  ...
out:
  ...
```